docs/src/tutorials/getting_started/transitioning_from_matlab.jl

```diff
@@ -0,0 +1,280 @@
+
+# ## Setting solver and options
+
+#  In order to set an optimizer with JuMP you can use at any point the command
+# `set_optimizer(model, Hypatia.Optimizer)`, where "Hypatia" is an example
+# solver. See the list of [Supported solvers](@ref) for more.
+
+# To configure the solver options you use the command `set_attribute(model,
+# "verbose", true)` after you set the optimizer, where as an example we were
+# setting the option verbose to true.
+#
+# A crucial difference is that with JuMP you always have to explicitly choose a
+# solver before optimizing. Both YALMIP and CVX allow you to leave it empty and
+# will try to guess an appropriate solver for the problem.
+
+# ## Optimizing
+
+# Like YALMIP, but unlike CVX, with JuMP you need to explicitly start the
+# optimization, with the command `optimize!(model)`. The exclamation mark here
+# is a Julia-ism that means the function is modifying its argument, `model`.
+
+# ## Extracting variables
+
+# Like YALMIP, but unlike CVX, with JuMP you need to explicitly ask for the value
+# of your variables after optimization is done, with the function call `value(x)`
+# to obtain the value of variable `x`. A subtlety is that unlike YALMIP the
+# function `value` is only defined for scalars, so for vectors and matrices you
+# need to use Julia broadcasting: `value.(v)`. There's also a specialized function
+# for extracting the value of the objective, `objective_value(model)`, which is
+# useful if your objective doesn't have a convenient expression.
+
+# ## Dual variables
+
+# Like YALMIP and CVX, JuMP allows you to recover the dual variables. In order
+# to do that, the simplest method is to name the constraint you're interested in,
+# for example, `bob = @constraint(model, sum(v) == 1)` and then, after the
+# optimzation is done, call `dual(bob)`.
+
+# See [Duals of variable bounds](@ref) for more.
+
+# ## Reformulating problems
+
+# Perhaps the biggest difference between JuMP and YALMIP and CVX is how far the
+# modeller is willing to go in reformulating the problems you give to it. CVX is
+# happy to reformulate anything it can, even using approximations if your solver
+# cannot handle the problem. YALMIP will only do exact reformulations, but is
+# still fairly adventurous, being willing to reformulate a nonlinear objective
+# in terms of conic constraints, for example. JuMP does no such thing: it only
+# reformulates objectives into objectives, and constraints into constraints, and
+# is fairly conservative at that. As a result, you might need to do some
+# reformulations manually, for which a good guide is available [here](@ref conic_tips_and_tricks).
+
+# An interesting possibility that JuMP offers is turning off reformulations
+# altogether, which will reduce latency if they're in fact not needed, or
+# produce an error if they are. To do that, set your optimizer with the option
+# `add_bridges = false`, as for example, in `set_optimizer(model, Hypatia.Optimizer, add_bridges=false)`
```

Comment:
We probably shouldn't tell people about this just yet. It's an extra for expert thing, not for new users.

Reply:
I had in mind users that are already experienced with optimization, just not JuMP and Julia. Feel free to remove it, though.

Reply:
Again, instead of removing we could just move it to the end in a "advanced" session

Reply:
I removed it. I don't want to add it to an advanced section. It doesn't belong in a "transitioning from MATLAB" tutorial.